test-cases/features/ics_space_amplification_goal_test.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 3600
+
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(factor=3) compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..250000000",
+                      "cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(factor=3) compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=250000001..500000000",
+                      "cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(factor=3) compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=500000001..750000000",
+                      "cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(factor=3) compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=750000001..1000000000"]
+
+stress_cmd: ["cassandra-stress write no-warmup cl=QUORUM n=1000000000 -schema 'replication(factor=3) compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..1000000000"]
+round_robin: true
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+ami_id_loader: 'ami-03c856a491257cb7f'
```

Comment:
@dkropachev is checking this for us again, let wait for his answer

Reply:
We had any answer from him ?

Reply:
no answer. sent him test log earlier.